src/addons/build.js

```diff
@@ -83,12 +88,12 @@ module.exports = function(XRegExp) {
      * var minutes = /^[0-5][0-9]$/;
      * var time = XRegExp.tag('x')`^ ${hours} (${minutes}) $`
      * time.test('10:59'); // -> true
-     * XRegExp.exec('10:59', time).minutes; // -> '59'
+     * XRegExp.exec('10:59', time)[1]; // -> '59'
```

Comment:
Maybe just remove this line and the capturing group around `minutes` since this is no longer showing an additional feature.